src/main/resources/lang/default.lang

```diff
@@ -1932,6 +1934,8 @@ types:
 	gamerule: gamerule¦s @a
 	attributetype: attribute type¦s @a
 	enchantmentoffer: enchantment offer¦s @a
```

Comment:
unless I'm just tired, shouldn't these ones be `@an` too?

(I know it's not related to this PR just asking about it since we're here)

Reply:
I can change it.